src/helpers/sql.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Query\Builder;
+if (!function_exists('dumpSql')) {
+    /**
+     * Print SQL query with bindings.
+     *
+     * @param Builder $builder
+     * @return string
+     */
```

Comment:
No other function file has a docblock. I assume this is because the documentation is in `readme.md`. You also must add this function in the documentation within `readme.md` (functions are documented in alphabetical order).